### ISSUE TITLE
Empty states

### ISSUE THREAD

**Issue by Misjohns** [instructlab/ui#96]
Provide empty states for the Submissions, Endpoints, and Chat pages.
@beaumorley

**Comment by Misjohns** [comment]
Empty state designs:
![image](https://github.com/user-attachments/assets/6fb74781-badb-43fc-acd9-56404f496198)
[PF Empty states](https://www.patternfly.org/components/table/react-demos/#empty-states)

**Comment by Misjohns** [comment]
### Empty state content

Icon: fa-search
Title: No results found
Description: Adjust your filters and try again.
Link button: Clear all filters

Icon: Spinner
Title: Loading

Icon: fa-exclamation-circle
Title: Unable to connect
Description: There was an error retrieving data. Check your connection and reload the page.

**Comment by zhuje** [comment]
I'd like to contribute, you can assign me. 

It seems like there are two parts to this task. 
1) Convert the current Dashboard Page from using Cards [[1](https://github.com/instructlab/ui/blob/67eefbbbea5b182bb2c2c97d1f5c8b2628afc77e/src/components/Dashboard/index.tsx#L178-L212)] to using the Patternfly Sortable Responsive Table [[2](https://www.patternfly.org/components/table/react-demos/#sortable---responsive)] 
2) Add Empty States to the Table (No results, Loading, and Unable to connect)

Is that correct?

References 
[1] https://github.com/instructlab/ui/blob/67eefbbbea5b182bb2c2c97d1f5c8b2628afc77e/src/components/Dashboard/index.tsx#L178-L212)
[2] https://www.patternfly.org/components/table/react-demos/#sortable---responsive

**Comment by Misjohns** [comment]
@zhuje Correct. Let me know if you'd prefer to break these into 2 separate issues.

**Comment by zhuje** [comment]
@Misjohns I think it probably would be better to break it up into 2 separate issues. I have a draft PR converting the Cards into a Table: https://github.com/instructlab/ui/pull/259 .

**Comment by vishnoianil** [comment]
Empty state is already implemented for the dashboard. New enhancement to the dashboard is captured through #489
